ui/pages/send/send-content/add-recipient/add-recipient.component.js

```diff
@@ -101,7 +104,7 @@ export default class AddRecipient extends Component {
 
     let content;
 
-    if (isValidAddress(query)) {
+    if (!isBurnAddress(query) && isValidHexAddress(query)) {
```

Comment:
It seems overly harsh to disallow differently-checksummed addresses here.

The same goes for the other uses of this function in the Send flow and the contacts flow.

Reply:
While that is true, it is the previously implemented behavior. Should I change that as part of this PR? These places used the previous 'isValidAddress' custom function which did check for mixed case and do validation using isValidChecksum.

Reply:
It would be perfectly reasonable to stick with targeting pre-v9.5.0 behavior IMO. Particularly since this is destined for a hotfix.